src/structures/Game.js

```diff
@@ -1,9 +1,13 @@
+const cdn = 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps';
+
 class Game {
 	constructor(game) {
+		this.name = game.name;
 		this.appID = game.appid;
 		this.playTime = game.playtime_forever;
-		this.playTime2 = game.playtime_2weeks || 0;
```

Comment:
Why was this removed?

Reply:
Sorry about that It should have been added back in, originally i thought it was absent from the api after adding the `include_appinfo` tag, because I tested a steam id with no playtime in the last 2 weeks. Ill add it back right now